docs/ml-features.md

```diff
@@ -1188,7 +1188,9 @@ categorical features. The number of bins is set by the `numBuckets` parameter. I
 that the number of buckets used will be smaller than this value, for example, if there are too few
 distinct values of the input to create enough distinct quantiles.
 
-NaN values: Note also that QuantileDiscretizer
+NaN values:
+NaN values will be removed from the column when `QuantileDiscretizer` fitting. This will produce
```

Comment:
"from the column when" -> "from the column during"
"model for making prediction and transformation" -> "model for making predictions"